### PR TITLE
Fix collaboration functionality

### PR DESCRIPTION
**Pull request by kusamkaur16**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -49,7 +49,10 @@ export default {
 </script>
 
 <style>
+<<<<<<< HEAD
+=======
 
+>>>>>>> 282b9f8702a5012350e6be0dcf922541fd4552d1
```
Comment (kkcmah) [inline src/App.vue]: lol remove merge conflict indicators

File: src/components/AccountModal.vue [hunk 2]
```diff
@@ -21,12 +21,19 @@
...
               </div>
             </div>
           </div>
+          <div class="modal-footer" id="update-footer">
+           <div class="container">
+             <button type="button" class="btn btn-outline-primary" id="change" data-toggle="modal" data-target="#loginModal">Change Email</button>
+             <button type="button" class="btn btn-outline-primary" id="change" data-toggle="modal" data-target="#loginModal">Change Username</button>
+             <button type="button" class="btn btn-outline-primary" id="change" data-toggle="modal" data-target="#loginModal">Change Password</button>
+           </div>
+         </div>
```
Comment (kkcmah) [inline src/components/AccountModal.vue]: duplicate of footer some lines down?

File: src/components/AccountModal.vue [hunk 3]
```diff
@@ -41,25 +48,70 @@
...
       username: '',
       compositions: [
-        {name: 'composition 1'},
-        {name: 'composition 2'},
-        {name: 'composition 3'},
-        {name: 'composition 4'}
-      ]
+        {name: 'test'}
+      ],
+      username: ''
```
Comment (kkcmah) [inline src/components/AccountModal.vue]: possibly duplicate 'username' data as above

File: src/components/Header.vue [hunk 4]
```diff
@@ -47,4 +51,37 @@ export default {
...
+        removeAll: this.username
+      }).then(function (result) {
+        that.$feathers.logout()
+      })
+
+      $('#loginModal').modal('show')
+    }
+  }
+}
+</script>
```
Comment (kkcmah) [inline src/components/Header.vue]: <script> section a duplicate from lines 19 - 32

Comment (kusamkaur16) [reply src/components/Header.vue]: fixed, but the comment no disappear

**Pull request merged**